app/server/lib/DocStorageManager.ts

```
@@ -234,6 +234,10 @@ export class DocStorageManager implements IDocStorageManager {
     // nothing to do
   }
 
+  public async wipeCache(): Promise<void> {
+    // nothing to do
```

Maybe this should throw an error saying it would be a bad idea to wipe a "cache" if it is actually the only copy :)

---

I'm also thinking that checking that a version of the doc exists in S3 could sometimes be life-saving :
- you're working on a document
- S3 storage is down, but you can still work on your document (in the background, snapshots don't trigger though)
- you wipe the "cache"
- all of your work is gone

---

> Maybe this should throw an error saying it would be a bad idea to wipe a "cache" if it is actually the only copy :)

I am not very comfortable with raising an exception, here are the reasons:
1. A cache may be many things, hence I feel like it is acceptable to just "do nothing" (just like the `flush` method) ;
2. this function could be reused elsewhere (I especially think of #1494), raising an exception would mean that the caller is aware of the nature of the storage management (local, S3, whatever), the inheritance is convenient for just allowing to ignore the call ;

An alternative I propose is otherwise that the `wipeCache` method is optional in the `IDocStorageManager` interface. The caller must check whether the method exists: use optional chaining or a condition + raise an exception if the method is not implemented (something that the endpoint could do). 

What do you think?